scripts/tigers.coffee

```diff
@@ -179,6 +179,20 @@ games = [{ date: Date.parse('4/6/15'), desc: "Twins 1:08p" },
 { date: Date.parse('10/3/15'), desc: "at White Sox TBD" },
 { date: Date.parse('10/4/15'), desc: "at White Sox TBD" }]
 
+daysOfWeek = ['Sunday','Monday','Tuesday','Wednesday','Thursday','Friday','Saturday']
+
+displayDate = (date) ->
+  now = Date.today().clearTime()
+  day = date.getDay()
+  dayOfWeek = daysOfWeek[day]
+  if Date.compare( date, now ) == 0
+    dayOfWeek = "Today"
+  else if day == now.getDay() + 1
+    dayOfWeek = "Tomorrow"
+  else if day <= now.getDay()
+    dayOfWeek = "Next #{dayOfWeek}"
```

Comment:
```getDay()``` is a range of 0..6.  So if today is Tuesday, ```now.getDay()``` is 2.

Since we filter out games that are no more than a week a way, the latest date can be is ```now.add(7).days()``` or "Next Tuesday".

So if we have Sunday as 0, Monday as 1 and Tuesday as 2, if their ```getDay()``` value is less than or equal to today, but the __date__ is not Today, than we know that they are next week and therefore I was using "Next #{}"

If we wanted to do "This #{}", we could either add an else statement, or we could flip it around and make it ```day > now.getDay()```

I was thinking of making this method only insert the day of week, if it is within "this week and next week" however, the filter we do in the respond method does that for us.